CHANGES.md

```diff
@@ -1,3 +1,28 @@
+# snare 0.4.0 (2020-xx-xx)
+
+## Breaking changes
+
+* The `email` option in `match` blocks has been replaced by the more generic
+  `errorcmd`. To obtain the previous behaviour:
```

Comment:
Did you want to mention the motivation? I think it's so that you can notify by other means if you so wish, right? Like slack / mattermost / irc / ...

Reply:
Is "more generic" not sufficient motivation? I'm happy to consider something else if you think not.

Reply:
Yeah OK.